src/stores/memory.ts

```diff
@@ -49,7 +49,7 @@ export function memoryStore(args?: MemoryConfig): MemoryStore {
     keys: async () => [...lruCache.keys()],
     mget: async (...args) => args.map((x) => lruCache.get(x)),
     async mset(args, ttl?) {
-      const opt = { ttl: ttl ? ttl : lruOpts.ttl } as const;
+      const opt = { ttl: ttl != null ? ttl : lruOpts.ttl } as const;
```

Comment:
It's `ttl ==! undefined`

Reply:
`ttl != null` equals`ttl !== undefined && ttl !== null`

https://github.com/lodash/lodash/blob/master/isNil.js#L19

Reply:
But by typing it is impossible for it to be null.